docs/usage/ObjectStorage.md

```diff
@@ -1,10 +1,29 @@
 ## Object Storage Setup
 
-CAM components use S3 object storage as temporary scratch space when performing migrations.  This storage can be any object storage that presents an `S3 like` interface.  Currently, we have tested AWS S3, Noobaa, and Minio.  
+CAM components use object storage as temporary scratch space when performing migrations. This document covers the process of setting up compatible Object Storage to use with CAM on the provider of your choice. The Object Storage credentials obtained from this document are needed for *configuring CAM Replication Repositories*.
 
-### Object Storage Setup with NooBaa
 
-NooBaa can run on an OpenShift cluster to provide an S3 compatible store for migration scratch space. We recommend loading NooBaa onto the destination cluster.
+### Supported Providers
+
+
+- **S3 Bucket** *(Any storage medium presenting an `S3 like` interface)*
```

Comment:
nit: `Any storage medium that exposes an S3 compatible endpoint`

Reply:
ACK